source/renderer/util/editor-tabs.js

```diff
@@ -29,7 +29,10 @@ module.exports = class EditorTabs {
     this._tippyInstances = []
 
     // Listen to the important events
-    this._div.onclick = (event) => { this._onClick(event) }
+    this._div.onclick = (event) => this._onClick(event)
+    // Listen for non-primary clicks
+    this._div.onauxclick = (event) => this._onClick(event)
```

Comment:
Please make sure to add the square brackets to these one-liners. I know they are not necessary, but it emphasises that we're dealing with a function here!

Reply:
Interesting, I would have argued that the square brackets indicate we're "ok" with putting logic related to the onclick up here, instead of indicating that the onclick logic should only reside in the lower _internal onClick.

But I'm just having a discussion, for the sake of discussion, I'll change it. 👍

Reply:
done.